src/components/common/MwButton/MwButton.tsx

```diff
@@ -0,0 +1,25 @@
+import './MwButton.scss';
+
+export interface MwButtonProps {
+  handleClick: Function;
+  buttonName?: string;
+  buttonClassName?: string;
+  children?: JSX.Element;
+}
+
+const MwButton = (props: MwButtonProps): JSX.Element => {
+  const { children, buttonName, buttonClassName, handleClick } = { ...props };
+
+  const handleChange = (): void => {
```

Comment:
You don't need this wrapper, can be written like so:
```
<button className={buttonClassName} onClick={onClick}>
```
Also, renaming from `onClick` to `handleClick` is not something that I've seen usually done, since it is increases the learning curve for someone who sees the code for the first time.
If you still want to keep the wrapper, the name could be something like `onClickInternal` or `_onClick`, so it doesn't deviate a lot from the action itself.

Reply:
will rename to onClickInternal